buildSrc/win.gradle

```diff
@@ -133,7 +137,9 @@ ext.WINDOWS_NATIVE_COMPILE_ENVIRONMENT = [
 ];
 def msvcVer = System.getenv("MSVC_VER") ?: "14.10.25017"
 def msvcBinDir = ""
-if (winVsVer == 150) {
+if (hasProperty('toolchainDir')) {
+    msvcBinDir = "$WINDOWS_VS_VSINSTALLDIR/VC/bin/${IS_64 ? 'x64' : 'x86'}"
```

Comment:
The msvcRedstDir will likely need similar treatment (i.e., a different directory when using toolchainDir).

Reply:
Exactly. I will fix it.